pkg/antctl/raw/multicluster/create/clusterclaim.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type clusterClaimOptions struct {
+	namespace  string
+	clusterSet bool
+}
+
+var clusterClaimOpt *clusterClaimOptions
+
+var clusterClaimExamples = strings.Trim(`
+Create a ClusterClaim in a leader or member cluster
+$ antctl mc create clusterclaim <CLUSTER_ID> -n <NAMESPACE> --clusterset 
+`, "\n")
+
+func (o *clusterClaimOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		return fmt.Errorf("the Namespace cannot be empty")
+	}
+
+	return nil
+}
+
+func NewClusterClaimCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "clusterclaim",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Create a ClusterClaim in a leader or member cluster",
+		Long:    "Create a ClusterClaim in a leader or member cluster",
+		Example: clusterClaimExamples,
+		RunE:    clusterClaimRunE,
+	}
+
+	o := &clusterClaimOptions{}
+	clusterClaimOpt = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterClaim")
+	command.Flags().BoolVarP(&o.clusterSet, "clusterset", "", false, "If present, a ClusterClaim of ClusterSet will be created")
```

Comment:
According to the user guide, we need both ClusterClaim and ClusterSet in both the leader and member clusters. And there are two kinds of ClusterClaim, one for cluster and another for ClusterSet. So I added a `clusterset` flag to distinguish them

Reply:
Then we need two commands to create the two claims? Could we use one to create both? I feel even better, we have one command to create the two Claims and ClusterSet together. If it is too big a change, we can do that in the next release.

Reply:
yes, @hjiajing is correct, leader and member need both ClusterClaim and ClusterSet. they need to use one ClusterClaim to announce its own Cluster ID with name `id.k8s.io`, and another one to announce the ClusterSet's ID with name 'clusterSet.k8s.io'.

Reply:
@jianjuns The two different claims have different metadata such name and value.  If we create them using one command, maybe the name will be the same. 
```bash
apiVersion: multicluster.crd.antrea.io/v1alpha1
kind: ClusterClaim
metadata:
  name: west-membercluster-id
  namespace: kube-system
name: id.k8s.io
value: test-cluster-west
---
apiVersion: multicluster.crd.antrea.io/v1alpha1
kind: ClusterClaim
metadata:
  name: clusterset-id
  namespace: kube-system
name: clusterSet.k8s.io
value: test-clusterset
```

Reply:
I did not get what you mean. Yes, they have different names and values, but why you cannot create them in one command?

We just need to add two parameters one like "--cluster-set <CLUSTER_SET_ID>", one like "--cluster-id <CLUSTER_ID>". The first Claim uses CLUSTER_ID as name, and the second uses CLUSTER_SET_ID as name.

Reply:
I changed my code. Then it will create two ClusterClaims in one command. One for the leader or member cluster and another for the ClusterSet. As for the name, `the metadata.Name` I use the `WellKnownClusterClaimClusterSet` and `WellKnownClusterClaimID` as we mentioned before.